Sources/HipMobileUI.iOS/Map/MapRenderer.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using CoreGraphics;
+using CoreLocation;
+using de.upb.hip.mobile.pcl.BusinessLayer.Models;
+using de.upb.hip.mobile.pcl.Common;
+using de.upb.hip.mobile.pcl.Helpers;
+using HipMobileUI.iOS.Map;
+using HipMobileUI.Map;
+using HipMobileUI.Navigation;
+using HipMobileUI.ViewModels.Pages;
+using MapKit;
+using UIKit;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.iOS;
+
+[assembly: ExportRenderer(typeof(OsmMap), typeof(MapRenderer))]
+
+namespace HipMobileUI.iOS.Map
+{
+    class MapRenderer : ViewRenderer<OsmMap, MKMapView> {
+
+        protected override void OnElementChanged(ElementChangedEventArgs<OsmMap> e)
+        {
+            base.OnElementChanged(e);
+           
+
+            if (Control == null)
+            {
+                var mapView = new MKMapView();
+                this.SetNativeControl(mapView);
+
+                var overlay = new MKTileOverlay ("http://tile.openstreetmap.org/{z}/{x}/{y}.png") {CanReplaceMapContent = true};
+                mapView.AddOverlay(overlay, MKOverlayLevel.AboveLabels);
+                mapView.OverlayRenderer = (view, mkOverlay) => new MKTileOverlayRenderer(overlay);
+
+                if (e.NewElement != null)
```

Comment:
Place this after the next if statement. Otherwise you might remove the wrong listenerns